integration_test/workflow_tests/run_tests.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+
+import testtools
+
+import test_packages
+
+
+class TracingStreamResult(testtools.StreamResult):
```

Comment:
A doc string might be good here especially pointing to the library maybe so we understand the dictionary and keys we are getting passed from status.

Reply:
Added